jobs/aws/eks-distro-build-tooling/alertmanager-postsubmits.yaml

```diff
@@ -47,12 +47,15 @@ postsubmits:
         command:
         - bash
         - -c
+        - env:
+          - name: PROJECT_PATH
+            value: 
```

Comment:
Also this is empty because you haven't set `projectPath` as a field in the minimal config, so empty value gets put in `jobConfig.ProjectPath` which is what is used in the `data` map in main.go. So each minimal job config needs to transform like this
**Before:**
```
commands:
- make something -C some_path
```

**After:**
```
projectPath: some_path
commands:
- make something -C $PROJECT_PATH
```